src/utils/testing/helpers.py

```diff
@@ -483,7 +483,7 @@ def create_editor_assignment(article, editor, **kwargs):
 
 def create_revision_request(article, editor, **kwargs):
     note = kwargs.get('note', 'Test note')
-    decision = kwargs.get('decision', review_models.review_decision()[0][0])
+    decision = kwargs.get('decision', review_models.ALL_REVIEW_DECISION_CHOICES[0][0])
```

Comment:
Where is `ALL_REVIEW_DECISION` defined? I think you used a function with lower case naming